pkg/relayer/session/sessiontree.go

```diff
@@ -240,9 +244,8 @@ func (st *sessionTree) Delete() error {
 
 	st.isClaiming = false
 
-	if err := st.treeStore.ClearAll(); err != nil {
-		return err
-	}
+	// We used to `st.treeStore.ClearAll()` here, but don't need to clean up the database, causing IO load,
+	// when we close the database and delete it from the disk right away.
```

Comment:
Would you mind making one more pass at this to improve readability. Perhaps something like:

```suggestion
	// NB: We used to `st.treeStore.ClearAll()` here which caused unnecessary I/O load;
	// however, since we close and delete the DB when ...(not sure when)..., there's 
	// no need to clean up here.
```

Reply:
Adopted Olshansk's wording